docs/make.bat

```diff
@@ -1,36 +1,36 @@
-@ECHO OFF
-
-pushd %~dp0
-
-REM Command file for Sphinx documentation
-
-if "%SPHINXBUILD%" == "" (
-	set SPHINXBUILD=sphinx-build
-)
-set SOURCEDIR=.
-set BUILDDIR=_build
-set SPHINXPROJ=reader
-
-if "%1" == "" goto help
-
-%SPHINXBUILD% >NUL 2>NUL
-if errorlevel 9009 (
-	echo.
-	echo.The 'sphinx-build' command was not found. Make sure you have Sphinx
-	echo.installed, then set the SPHINXBUILD environment variable to point
-	echo.to the full path of the 'sphinx-build' executable. Alternatively you
-	echo.may add the Sphinx directory to PATH.
-	echo.
-	echo.If you don't have Sphinx installed, grab it from
-	echo.http://sphinx-doc.org/
-	exit /b 1
-)
-
-%SPHINXBUILD% -M %1 %SOURCEDIR% %BUILDDIR% %SPHINXOPTS%
-goto end
-
-:help
-%SPHINXBUILD% -M help %SOURCEDIR% %BUILDDIR% %SPHINXOPTS%
-
-:end
-popd
+@ECHO OFF
```

Comment:
This shouldn't change. I'm guessing line endings changed, due to some unexpected interaction with your git config (or maybe your editor normalized newlines for some reason).

If it's git, in theory adding a `*.bat -text` line to https://github.com/lemon24/reader/blob/402d50dc7faf8a40ffb3485a0be3f99f4fa8e1ab/.gitattributes#L3 should fix it.

Reply:
Added https://github.com/lemon24/reader/commit/25a7207edac09d1e9e78d2864a640e02738c9904 to prevent this from happening in the future (but you still need to rebase and revert this one file).

Reply:
I'm not sure what's happening here, on my branch at Sakunam/reader/issue-317, it shows that the make.bat is much removed all the old text but then added it all back? Here, it's just showing what it showed before so I'm not entirely sure what to do.

Reply:
OK, here's my best attempt to explain what happened (still not confident it's 100% accurate):

Before any changes:

```
$ git checkout 3.12 -f 
HEAD is now at fed7a51 Release 3.12.
$ git ls-files --eol | grep make.bat
i/crlf  w/crlf  attr/text eol=lf      	docs/make.bat
------- ------  ------------------   
 ^       ^       ^         
 |       |       '- eolattr; lf means "same line endings in working dir as in index"
 |       '- w/eolinfo aka line endings in the working tree
 '- i/eolinfo aka line endings in the index
```

You then changed the line endings for make.bat from CRLF to LF, likely because you had [global core.autocrlf true](https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_formatting_and_whitespace) ([more details](https://stackoverflow.com/a/2825829)) aka "auto-convert CRLF line endings into LF when adding a file to the index" *before* I commited 25a7207 (ignore the fact that f8833db comes after 25a7207, in the original sequence of changes in the PR, the make.bat change came before):

```
$ git checkout f8833db -f
HEAD is now at f8833db Add .reader.mark-as-read.once. #317 #333
$ git ls-files --eol | grep make.bat
i/lf    w/crlf  attr/text eol=crlf    	docs/make.bat
----
 ^
 '- changed
```

25a7207 follows (technically before f8833db, practically after the make.bat change):

```
$ git checkout 25a7207 -f 
HEAD is now at 25a7207 .gitattributes: always use crlf line endings for bat files.
$ git ls-files --eol | grep make.bat
i/crlf  w/crlf  attr/text eol=crlf    	docs/make.bat
                ------------------   
                 ^         
                 '- crlf means "set line endings to CRLF in working dir"
```

Now, I don't really know how to change the index line endings back to CRLF, but thanks to 25a7207, it doesn't really matter, they'll always be CRLF when checked out.

Reply:
OK, I managed to revert the line endings change in https://github.com/lemon24/reader/commit/48c6bebc93e2c82471fad40ec2a9bdf8c5a1eb84:

```
$ git ls-files --eol | grep make.bat
i/crlf  w/crlf  attr/text eol=crlf    	docs/make.bat
```

Here's how, in case anyone stumbles upon this:

* change .gitattributes `*.bat text eol=crlf` to `*.bat -text` (aka treat as binary); commit
* fix make.bat line endings in editor; commit amend
* change .gitattributes  back to `*.bat text eol=crlf`; commit amend